### PR TITLE
Created generic_meetup_help_letter.md

### PR DESCRIPTION
**Pull request by UwanaIkaiddi**
Initial draft for Generic Meetup Help Letter

### CODE REVIEW

**Comment by mjang** [comment]
Question -- where would you link this? (I'd add a link in this page: http://www.writethedocs.org/organizer-guide/meetups/starting/ )

**Comment by plaindocs** [comment]
Sounds good to me.

File: docs/organizer-guide/meetups/generic_meetup_help_letter.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Generic letter asking for help with Meetup 
+
+**Subject:** Help with WriteTheDocs
+
+Greetings!
+
+We'd like invite you to join us more actively this coming year!
+
+By helping out, you'll: 
+* learn more about the [WriteTheDocs](http://www.writethedocs.org/) organization
```
Comment (meker12) [inline docs/organizer-guide/meetups/generic_meetup_help_letter.md]: On the [main website](http://www.writethedocs.org/), the spelled out organization name is "Write the Docs" -- I think the link text will look better with the spaces.

File: docs/organizer-guide/meetups/generic_meetup_help_letter.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* have the opportunity to get more involved in the larger US and international WriteTheDocs community
+* network with other like-minded professionals
+* raise the visibility of the latest techniques and technologies for creating great documentation
+
+Our WriteTheDocs Meetup group is fortunate to have an active community, experienced organizers, and backing from the WriteTheDocs organization.
```
Comment (meker12) [inline docs/organizer-guide/meetups/generic_meetup_help_letter.md]: Our Write the Docs ...   backing from the Write the Docs organization

File: docs/organizer-guide/meetups/generic_meetup_help_letter.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Our current organizer(s) spend a few days a month brainstorming ideas and planning the events. We would need your help with:
```
Comment (meker12) [inline docs/organizer-guide/meetups/generic_meetup_help_letter.md]: Suggested revision:

We would need your help ==> We are asking for your help with the planning or other activities like the following:

File: docs/organizer-guide/meetups/generic_meetup_help_letter.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+* networking with other tech and Meetup groups
+* finding interesting content
+* recruiting speakers
+* occasionally hosting Meetup events
+
+Helping out is a great opportunity for everyone. If you're new to the field, it's a great way to learn, network, and develop new skills. For more experienced documentarians, it's an opportunity to share you experciences, refresh some skillsets, and discuss challenges with other writers.
```
Comment (meker12) [inline docs/organizer-guide/meetups/generic_meetup_help_letter.md]: share you experciences, refresh some skillsets ==>  share your stories and expertise, refresh your skills, and discuss challenges with other writers.

**Review by meker12 — COMMENTED** [review]
Thanks for adding this, @uwikaiddi and @mjang . Nice for others to have a starter template for reaching out. I know it worked for the ATX meetup :-)

File: docs/organizer-guide/meetups/generic_meetup_help_letter.md (hunk 1 above)
Comment (plaindocs) [reply docs/organizer-guide/meetups/generic_meetup_help_letter.md]: :+1:

**Comment by mjang** [comment]
Merging -- will add a link in a separate PR.

**Pull request merged**